src/ui/shared/clock/clocks/BronsteinClock.ts

```diff
@@ -18,6 +18,8 @@ export default function BronsteinClock(time: number, increment: number, onFlag:
     isRunning: false
   }
 
+  const id = Math.random()
```

Comment:
`Math.random()` makes not a so good id. Prefer `performance.now()` which is guaranteed to be unique.

Also, I don't see why do you use the id?